pkg/operator/ceph/cluster/osd/health.go

```diff
@@ -94,6 +97,32 @@ func (m *OSDHealthMonitor) Update(removeOSDsIfOUTAndSafeToRemove bool) {
 
 // checkOSDHealth takes action when needed if the OSDs are not healthy
 func (m *OSDHealthMonitor) checkOSDHealth() error {
+	err := m.checkByOSDDump()
+	if err != nil {
+		return err
```

Comment:
Instead of returning an error, let's just log it and continue. If we returned the error, we might not ever get to check the device classes.

Reply:
fixed